python/pom.xml

```diff
@@ -125,6 +125,8 @@
               <target>
                 <unzip src="${project.build.directory}/../../interpreter/python/py4j-${python.py4j.version}.zip"
                        dest="${project.build.directory}/../../interpreter/python"/>
+                <zip destfile="${project.build.directory}/../../interpreter/python/py4j-${python.py4j.version}-src.zip"
```

Comment:
It was existing bug in python Interpreter. That codes didn't works as expected. So I changed it to normal